sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/compatibility/SpringCloudAzureCompatibilityNotMetException.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.autoconfigure.compatibility;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Create CompatibilityNotMetException for CompatibilityNotMetFailureAnalyzer
+ */
+public class SpringCloudAzureCompatibilityNotMetException extends RuntimeException {
+    final List<VerificationResult> results;
+
+    /**
+     * Constructor of CompatibilityNotMetException
```

Comment:
Update the name of the class of SpringCloudAzureCompatibilityNotMetException in the javadoc too